### PR TITLE
Runescape duplicate removals

### PR DESCRIPTION
**Pull request by N0ZTRA**
Remove the entry behind the three ironman icons, essentially duplicate the circles but with less info. Links are the same.

Remove the less informationally useful Crab Rave $11 entry, no link difference.

Also remove that one grey pixel entry because it is rather extraneous.

### CODE REVIEW

File: web/atlas.json [hunk 1]
```diff
@@ -1059,7 +1059,6 @@
...
 {"id": "twqx4v", "submitted_by": "oski165", "name": "Helltaker", "description": "A picture of a character from a small Polish videogame Helltaker named Lucifer along with a small logo", "website": "", "subreddit": "", "center": [1990.5, 1254.5], "path": [[1981.5, 1237.5], [1981.5, 1271.5], [1998.5, 1271.5], [1999.5, 1237.5]]},
-{"id": "twqx0n", "submitted_by": "FakeGoonmachine", "name": "Ironman Mode", "description": "These three helmets represent 3 'Ironman' game modes within RuneScape: Standard, Hardcore, and Ultimate. Ironman Mode is an account type that requires the player to be entirely self-sufficient. Ironmen are locked out of or restricted in most forms of interactions with other players such as trading, the Grand Exchange, PvP, most group minigames, and almost all other group activities.", "website": "https://oldschool.runescape.wiki/w/Ironman_Mode", "subreddit": "/r/ironscape, /r/2007scape", "center": [90.5, 42.5], "path": [[63.5, 50.5], [63.5, 34.5], [117.5, 34.5], [117.5, 50.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:1062]: This is valid as an encompassing entry for all three Ironmade modes.

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]
Address comments.

File: web/atlas.json (hunk 1 above)
Comment (N0ZTRA) [reply web/atlas.json:1062]: Sounds fine, no need to implement this commit.

**Comment by N0ZTRA** [comment]
Edit unnecessary.

**Pull request closed without merging**